style.css

```diff
@@ -0,0 +1,102 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  min-height: 100vh;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  
+}
+
+.container-top h1 {
+    color: #FFFFFF;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    font-family: 'Roboto';
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 24px;
+    height: 24px;
+    margin: 24px;
+}
+
+.container h3 {
+  /* width: 327px;
+  height: 27px; */
+  color: #FFF5E1;
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  margin: 24px;
+}
+
+.container p {
+  color: #FFFFFF;
+  font-family: 'Poppins';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  margin: 24px;
+}
+
+
+.container-top {
+  background: #3C3A39;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  text-align: center;
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  left: 0px;
+  top: 0px;
+}
+
+.container img {
+    margin: 24px;
+}
+
+
+
+.primary-text {
+  background-color: #1C1A19;
+  position: absolute;
+  width: 375px;
+  height: 590px;
+  left: 0px;
+  top: 92px;
+}
+
+.primary-text h2 {
+  color: #FF6B00;
+  font-family: 'Crete Round';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  width: 327px;
+  height: 20px;
+  margin: 120px 24px;
+}
+
+.primary-text .social-media-icon {
+    margin-bottom: 6rem; 
+}
```

Comment:
- [ ] As stated the project requirement You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1), 
- [ ] So Kindly make sure that headline section has the proper background image (decorations are always background images, not `img` tags).If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.
- [ ] Kindly make sure that The section has the margins (top, bottom, left, right) defined in the design. So Kindly adjust bottom margins just like the figma design.

The Figma design Look look likes this  : 

![Capture](https://user-images.githubusercontent.com/93598189/204980905-d778d2d4-7e71-406b-b9ba-8cfe9799789c.PNG)

Your Project design Look likes this : 
 
![Capture](https://user-images.githubusercontent.com/93598189/204981152-0a6e1f71-bd95-423a-b681-3fd79f3767e9.PNG)

Reply:
Hello

Please I have made the necessary correction as pointed out. The challenge is the linters work perfectly well on my terminal but not on my Github. What do you suggest I do to fix it?

Reply:
@RitaDee, I think you can try deleting the `node modules folder`, `package-lock.json` file, and all the linter files (`.hintrc`, `.stylelintrc.json`, `linters.yml`). Then try re-installing them from the beginning. I hope it will not take that much time, but it can solve the issue maybe.